### PR TITLE
Remove systemd

### PR DESCRIPTION
**Pull request by robbe-haesendonck**
Removed systemd from open-balena-vpn container to allow running the container unprivileged.
Change-type: minor

---
##### Contributor checklist
- [ ] Build output has been rebuilt and tested
- [ ] Introduces security considerations
- [ ] Tests are included
- [ ] Documentation is added or changed
- [x] Affects the development, build or deployment processes of the component

##### Reviewer Guidelines
- When submitting a review, please pick:
  - '*Approve*' if this change would be acceptable in the codebase (even if there are minor or cosmetic tweaks that could be improved).
  - '*Request Changes*' if this change would not be acceptable in our codebase (e.g. bugs, changes that will make development harder in future, security/performance issues, etc).
  - '*Comment*' if you don't feel you have enough information to decide either way (e.g. if you have major questions, or you don't understand the context of the change sufficiently to fully review yourself, but want to make a comment)
---

### CODE REVIEW

**Comment by robbe-haesendonck** [comment]
Note: capability NET_ADMIN is still required.

**Comment by ab77** [comment]
@robbe-haesendonck we still require Systemd variant of this component, as it is used in local-dev environments on top of mDNS. If you could update the PR to build both in parallel (using .resinci.yml), it should be OK to merge.

**Comment by robbe-haesendonck** [comment]
@ab77 Is there documentation available for the resinci.yml file?

**Comment by ab77** [comment]
> @ab77 Is there documentation available for the resinci.yml file?

unfortunately not, but for your purposes, something like [this](https://github.com/balena-io-modules/open-balena-base/blob/master/.resinci.yml) should work:

```
---
reviewers: 1
docker:
  builds:
    - path: .
      dockerfile: Dockerfile
      docker_repo: balena/open-balena-vpn
    - path: .
      dockerfile: Dockerfile.no-systemd
      docker_repo: balena/open-balena-vpn:no-systemd
```

**Comment by robbe-haesendonck** [comment]
@ab77 I changed the resinci file as requested and moved the no-systemd version to Dockerfile.no-systemd

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM balena/open-balena-base:v13.4.0 as base
```
Comment (ab77) [inline Dockerfile:1]: @robbe-haesendonck I don't believe this and most of the below changes are intentional?

**Review by ab77 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (robbe-haesendonck) [reply Dockerfile:1]: Yeah, I don't exactly know what went wrong. Be it in my mind.
I'll check/rollback what's necessary when I've got the time.

**Pull request closed without merging**